### ISSUE TITLE
Catalog grids need some UX work

### ISSUE THREAD

**Issue by NateWr** [pkp/pkp-lib#1333]
The new catalog grids (Catalog, Categories, Series, Spotlights) are great and a huge improvement over what we had before.

Playing around with them today I had a few UX issues that I'd like to sort out for future releases (after OMP 1.2).
- [ ] The layout of the search filters and submit button could be improved for clarity. I'd like to see the text search field on the left and, where it exists, the category/series dropdown on the right, so that we mimic the look of the Users grid.
- [ ] When we update from the filters, I'd like to see some kind of visual indication that loading is in process.
- [ ] The practice of updating the publication list as soon as an option from the dropdown lists is selected will present accessibility issues for people using screen readers. Many screen readers must _select_ an option before reading it.
- [ ] The ordering of features could lead to some confusion as it is. I'd like to see:
  - [ ] The language on the link become "Order Features"
  - [ ] The `.order_message` hover tip should ideally be replaced by a fully-grayed and disabled row for items that can't be ordered. If a hover tip is kept let's perform the show/hide via CSS.
  - [ ] It would be nice to find some way to only show ordering when features are selected. Or maybe update the list to show features only once the ordering is initiated.

**Comment by beghelli** [comment]
Hi @NateWr , thanks for the review. :)
Item two is already happening, but the spinner is next to the search button, which might be overlooked.
Last item is easy to do, I can try to implement that after finishing my last bug.

**Comment by NateWr** [comment]
Hey @beghelli, thanks! I probably just didn't notice the loading spinner in my local dev cause it's loading so quick.

To be honest, I think this none of this is critical and it can probably be postponed to after OMP 1.2. I don't want to derail you from more important work you may have ahead. I just wanted to log it while I was looking at it.

**Comment by NateWr** [comment]
This is obsolete as we use a new ListPanel for the catalog entries in OMP 3.1.
